numpy/core/src/umath/loops.c.src

```diff
@@ -1431,23 +1501,53 @@ TIMEDELTA_mm_m_remainder(char **args, npy_intp const *dimensions, npy_intp const
 NPY_NO_EXPORT void
 TIMEDELTA_mm_q_floor_divide(char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
 {
-    BINARY_LOOP {
-        const npy_timedelta in1 = *(npy_timedelta *)ip1;
+    /* TODO: This code is similar to array floor divide*/
```

Comment:
Lets just make it a NOTE? Probably just me, though... (otherwise same comments as above apply of course).